Source/WebCore/rendering/RenderTable.cpp

```diff
@@ -1508,9 +1508,9 @@ RenderTableCell* RenderTable::cellAfter(const RenderTableCell* cell) const
 
 LayoutUnit RenderTable::baselinePosition(FontBaseline baselineType, bool firstLine, LineDirectionMode direction, LinePositionMode linePositionMode) const
 {
-    return valueOrCompute(firstLineBaseline(), [&] {
-        return RenderBox::baselinePosition(baselineType, firstLine, direction, linePositionMode);
-    });
+    if (auto baselinePos = firstLineBaseline())
+        return (direction == HorizontalLine ? marginTop() : marginRight()) + firstLineBaseline().value();
```

Comment:
I think you want to change "firstLineBaseline().value()" to just "baselinePos.value()", since you just captured that variable in line 1511. No need to call that accessor again.

Or, you could do:

```
if (auto baselinePos = firstLineBaseline())
    return (direction == HorizontalLine ? marginTop() : marginRight()) + *baselinePos;
```

Reply:
Whoops yes thank you. Changed.